resources/js/components/dashboard/ControlGroup.vue

```diff
@@ -1,7 +1,19 @@
 <template>
-  <div class="control-group container">
-    <h1>Control Group</h1>
+<div class="control-group container">
+  <div class="row">
+    <div class="col-8 offset-2 text-center">
+      <div class="control-group__body">
+        <i class="control-group__icon fas fa-user" style="color:#299afb"></i>
+        <div class="control-group__content">
```

Comment:
Since the body text is technically one paragraph, let's try and keep it inside one <p> tag and use <br> to break up the lines according to our wireframe.

**Recommendation:**
`          <p class="control-group__text">Thank you for participating.<br>
          You are part of the control group.<br>
          You will receive an email with further<br>
          instructions.</p>
        </div>
`

Reply:
Thank you!